### ISSUE TITLE
[Feature]: A way to install with a seperate installer

### ISSUE THREAD

**Issue by tekigg** [illixion/vscode-vibrancy-continued#117]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe.

I'm trying to install [VScode animation](https://marketplace.visualstudio.com/items?itemName=BrandonKirbyson.vscode-animations) alongside Vibrancy and I tried importing it using the built in import config of VScode. Unfortunately it doesn't seem to work, plus it's a little limited it seems.

### Describe your idea or feature request

Is there a way to get the script path of VScode Vibrancy and import it with something like [Apc Customize UI++](https://marketplace.visualstudio.com/items?itemName=drcika.apc-extension) ?

**Comment by illixion** [comment]
Vibrancy makes changes to `/vs/code/electron-sandbox/workbench/workbench.html`, `/main.js` and on macOS, `/vs/code/electron-main/main.js` in VSCode's installation folder. You'll also need to copy the contents of `runtime` from this repo to `/vscode-vibrancy-runtime-v6`. You can likely perform a diff on those files before and after installing Vibrancy to get the full list of changes made.

You can see the code that makes these changes here: https://github.com/illixion/vscode-vibrancy-continued/blob/master/extension/index.js

Vibrancy works with any other well-written extensions that don't outright replace the contents of VSCode's internal files, but rather patch them by making only the necessary changes. If that's not the case, then there's no way to guarantee that Vibrancy won't cause VSCode to break or that the changes will remain revertible. This check is done by searching for certain HTML code that should always be present in workbench.html, so you should ensure that it's not being completely overwritten by either of those extensions.

Finally, I've checked the repo of the APC project, and it appears that they have their own version of a transparency effect that you could try using: https://github.com/drcika/apc-extension/issues/13#issuecomment-1596221902

**Comment by amnweb** [comment]
No, it’s not the same. APC can make the window transparent but can’t add blur. So, because of all this I have combined vibrancy + apc and I got some interesting result on windows 11 :D

Anyway both extension can work together only when i set titlebar to native for some reason.

![Screenshot 2023-12-21 021234](https://github.com/illixion/vscode-vibrancy-continued/assets/16545063/b0bba2ac-5783-400e-aa4a-261774a68fde)

**Comment by RoyalFoxy** [comment]
@amnweb I don't think you read the discussion illixion regarding APC and vibrancy cause I could make it work easily **and it is vibrancy, not just transparent**.

**Comment by RoyalFoxy** [comment]
I just tried it on vscode insiders and it only makes the entire window background transparent as vibrancy doesn't work but that can't be your issue cause vibrancy-continued wouldn't work either...

**Comment by tekigg** [comment]
> Vibrancy makes changes to `/vs/code/electron-sandbox/workbench/workbench.html`, `/main.js` and on macOS, `/vs/code/electron-main/main.js` in VSCode's installation folder. You'll also need to copy the contents of `runtime` from this repo to `/vscode-vibrancy-runtime-v6`. You can likely perform a diff on those files before and after installing Vibrancy to get the full list of changes made.
> 
> You can see the code that makes these changes here: https://github.com/illixion/vscode-vibrancy-continued/blob/master/extension/index.js
> 
> Vibrancy works with any other well-written extensions that don't outright replace the contents of VSCode's internal files, but rather patch them by making only the necessary changes. If that's not the case, then there's no way to guarantee that Vibrancy will work and won't cause VSCode to break. This check is done by searching for an HTML CSP tag that should always be present in workbench.html (and that we need to modify), so you should ensure that it's not being completely overwritten by either of those extensions.
> 
> Finally, I've checked the repo of the APC project, and it appears that they have their own version of a transparency effect that you could try using: [drcika/apc-extension#13 (comment)](https://github.com/drcika/apc-extension/issues/13#issuecomment-1596221902)

Thank you so much for taking in the time to look into this! 

I've been trying to get vibrancy working on that end but it doesn't seem to be working on Windows, although that's an APC issue rather than vsc vibrancy. Or maybe I'm just not figuring out the correct configuration as I only found examples for OSX :c

**Comment by tekigg** [comment]
> I just tried it on vscode insiders and it only makes the entire window background transparent as vibrancy doesn't work but that can't be your issue cause vibrancy-continued wouldn't work either...

That's my problem. I can't seem to be able to have both working at once without one of them breaking T-T
I'm trying to have [vscode-animations ](https://marketplace.visualstudio.com/items?itemName=BrandonKirbyson.vscode-animations) and no matter what I do either both break or one of them breaks depending on the install order-

**Comment by amnweb** [comment]
> I just tried it on vscode insiders and it only makes the entire window background transparent as vibrancy doesn't work but that can't be your issue cause vibrancy-continued wouldn't work either...


![Screenshot 2023-12-21 104055](https://github.com/illixion/vscode-vibrancy-continued/assets/16545063/5e342411-453b-405a-a938-2cfc9d6d45af)

**Comment by tekigg** [comment]
kinda-maybe-perhaps-off-topic but @amnweb can I have your configs? :0

**Comment by RoyalFoxy** [comment]
@MrOxMasTer have you tried this

```json
{
  "animations.Install-Method": "Apc Customize UI++",
  "apc.electron": {
    "backgroundColor": "#0000",
    "frame": false,
    "transparent": true,
    "vibrancy": "under-window",
    "visualEffectState": "active",
  },
  "apc.imports": [
    "file:///Users/royalfoxy/.vscode/extensions/brandonkirbyson.vscode-animations-2.0.1/dist/updateHandler.js"
  ],
}
```

Note that this only works on macos as `vibrancy` is their window blur. I couldn't find anything on their github page about Windows' Mica and Acrylic if you are on a windows machine.
